components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -31,248 +27,273 @@
+        <li>Still having trouble? Consult your network administrator or Internet provider for assistance.</li>
+      </ul>
      ]]></string>
 
+    <!-- The document title for an error page (i.e., the contents of the `<title>` tag). -->
     <string name="error_unknown_proxy_host_title">Proxy Server Not Found</string>
     <string name="error_unknown_proxy_host_message"><![CDATA[
-<p>The browser is configured to use a proxy server, but the proxy could not be found.</p>
-<ul>
-<li>Is the browser\’s proxy configuration correct? Check the settings and try again.</li>
-<li>Is the computer connected to an active network?</li>
-<li>Still having trouble? Consult your network administrator or Internet provider for assistance.</li>
-</ul>
+      <p>The browser is configured to use a proxy server, but the proxy could not be found.</p>
+      <ul>
+        <li>Is the browser’s proxy configuration correct? Check the settings and try again.</li>
+        <li>Is the computer connected to an active network?</li>
+        <li>Still having trouble? Consult your network administrator or Internet provider for assistance.</li>
+      </ul>
      ]]></string>
 
-
+    <!-- The document title for an error page (i.e., the contents of the `<title>` tag). -->
     <string name="error_safe_browsing_malware_uri_title">Malware site issue</string>
     <!-- The %1$s will be replaced by the malicious website URL-->
     <string name="error_safe_browsing_malware_uri_message"><![CDATA[
-<p>The site at %1$s has been reported as an attack site and has been blocked based on your security preferences.</p>
+      <p>The site at %1$s has been reported as an attack site and has been blocked based on your security preferences.</p>
      ]]></string>
 
+    <!-- The document title for an error page (i.e., the contents of the `<title>` tag). -->
     <string name="error_safe_browsing_unwanted_uri_title">Unwanted site issue</string>
     <!-- The %1$s will be replaced by the malicious website URL-->
     <string name="error_safe_browsing_unwanted_uri_message"><![CDATA[
-<p>The site at %1$s has been reported as serving unwanted software and has been blocked based on your security preferences.</p>
+      <p>The site at %1$s has been reported as serving unwanted software and has been blocked based on your security preferences.</p>
      ]]></string>
 
+    <!-- The document title for an error page (i.e., the contents of the `<title>` tag). -->
     <string name="error_safe_harmful_uri_title">Harmful site issue</string>
     <!-- The %1$s will be replaced by the malicious website URL-->
     <string name="error_safe_harmful_uri_message"><![CDATA[
-<p>The site at %1$s has been reported as a potentially harmful site and has been blocked based on your security preferences.</p>
+      <p>The site at %1$s has been reported as a potentially harmful site and has been blocked based on your security preferences.</p>
      ]]></string>
```

Comment:
while you are fixing the indents, all of these are off by a single space

Reply:
I wish our formatter handled these CDATA tags so we didn't have to do it by hand - fixed!